package.json

```diff
@@ -15,16 +15,34 @@
       "react"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
```

Comment:
We don't want to lint JSON files or any other files?

Reply:
@popojargo thats up to you guys ... if there is a lot of JSON, sure I can add that in.